airbyte-integrations/connectors/destination-snowflake/src/main/java/io/airbyte/integrations/destination/snowflake/SnowflakeCopyS3Destination.java

```diff
@@ -28,7 +29,7 @@ public AirbyteMessageConsumer getConsumer(final JsonNode config,
         getDatabase(config),
         getSqlOperations(),
         getNameTransformer(),
-        getS3DestinationConfig(config),
+        getS3CopyConfig(config),
```

Comment:
I suspect you need to use `getS3CopyConfig(config.get("loading_method"))` here. But it's worth running acceptance tests to verify that.

Reply:
Updated, thanks